tasks/request_files/API.md

```diff
@@ -15,7 +15,7 @@
 # request\_files
 
 Name: request_files.py
-Description:  Lambda function that makes a restore request from glacier for each input file.
+Description:  Lambda function that makes a restore request to archive for each input file.
```

Comment:
Maybe `to the archive` instead of `to archive`?

Reply:
Changed